manifests/profile/www_lib/vhosts/deepblue.pp

```diff
@@ -74,6 +74,11 @@
         rewrite_cond => '%{REQUEST_URI} !^((\/?|/index.html)$|/splash/|/cosign/valid)',
         rewrite_rule => '^(.*)$	http://bulleit-2.umdl.umich.edu:8080$1 [P]'
       },
+      {
+        comment      => 'Deep Blue Preservation redirect',
+        rewrite_cond => '^(.*)$ https://deepblue.lib.umich.edu/static/about/deepbluepreservation.html [P]',
```

Comment:
We won't need a condition here, just the path portion moved to the pattern in the rule.

Reply:
Got it, change rewrite_cond to path....